packages/@lwc/engine-core/src/framework/main.ts

```diff
@@ -51,3 +50,46 @@ export type {
     WireAdapterConstructor,
     WireAdapterSchemaValue,
 } from './wiring';
+
+export {
```

Comment:
Can we add a note here saying that all these methods are intended to be used for packages overriding the dom mechanism? Also, can we use `export * from` instead here?

Reply:
I can add a note. But I don't think we can `export *` because the `renderer.ts` exposes plenty of APIs that are only used internally to `engine-core` that we don't want exposed (`getAttribute`, `hasAttribute`, etc.).